tests/cli/test_buildspec.py

```diff
@@ -136,6 +136,28 @@ def test_buildspec_find_invalid():
     cache.print_invalid_buildspecs(error=False)
 
 
+@pytest.mark.cli
+def test_buildspec_find_invalid_terse():
+
+    cache = BuildspecCache(configuration=configuration, terse=True)
+    cache.print_invalid_buildspecs(error=True)
+    cache.print_invalid_buildspecs(error=False)
+
+
+@pytest.mark.cli
+def test_buildspec_find_invalid_terse_no_header():
+
+    cache = BuildspecCache(configuration=configuration, terse=True, header=False)
+    cache.print_invalid_buildspecs(error=True)
+    cache.print_invalid_buildspecs(error=False)
+
+
+@pytest.mark.cli
+def test_buildspec_find_invalid_terse_error():
+    cache = BuildspecCache(configuration=configuration, terse=True, header=False)
+    cache.print_invalid_buildspecs(error=True)
+
+
```

Comment:
@JeffreyFG can you please rewrite this into just one method and call `BuildspecCache` once and then just invoke `cache.print_invalid_buildspecs` for all the different option. 

Note that you can pass in option like `terse` and `header` as follows which will take value from argument instead of `self.terse`. This also avoids having to call the **__init__** method of BuildspecCache multiple times which can slow down performance of regression test

```
cache.print_invalid_buildspecs(terse=True, header=True, error=True)
cache.print_invalid_buildspecs(terse=True, header=True, error=False)
```